samples/bluetooth/direct_test_mode/README.rst

```diff
@@ -117,9 +117,20 @@ The RADIO can control up to 8 GPIO pins for the purpose of controlling the exter
 
 The antenna is chosen by writing consecutive numbers to the SWITCHPATTERN register.
 This means that the antenna GPIO pins act like 8-bit registers.
-In other words, for the first antena, antenna pin 1 is active, for the second antenna, pin 2 is active, for the third antenna, pins 1 and 2 are active, and so on.
+In other words, for the first antenna, antenna pin 1 is active, for the second antenna, pin 2 is active, for the third antenna, pins 1 and 2 are active, and so on.
 
-Vendor-Specific packet payload
+nRF21540 Front-End-Module
+=========================
```

Comment:
We could probably reuse the FEM support stuff from the Thread CLI sample, as we do in Thread CoAP Client sample. If the text there is not good as such, we might want to make this text close to that one, and then reuse it in the Radio test sample.

Reply:
We could use a documentation from the Zigbee PR at this moment https://github.com/nrfconnect/sdk-nrf/pull/3688/files. There is a reference to MPSL nrf21540 driver which isn't used here but I think this should not bother us.

Reply:
We have the PR #3688 merged now, and it contains a generic sample_fem_support.txt. Is it generic enough to be used in these docs?

Reply:
If the sample_fem_support.txt is not suitable to be used in these samples, we may need to create a suitable text to one of the samples and then use it (by include) in the other one. Something like:

.. include:: samples/bluetooth/direct_test_mode/README.rst
   :start-after: FEM_support_start
   :end-before: FEM_support_end_end

Reply:
At this moment we can use these generic doc. I modified samples documentation